api/Window.json

```diff
@@ -1163,55 +1163,6 @@
           }
         }
       },
-      "clipboardchange_event": {
```

Comment:
I can find no trace of this event in Chromium, which is where it might have been given the maintainers of https://w3c.github.io/clipboard-apis/#clipboard-event-clipboardchange.